### PR TITLE
Mvs schemazen create tables

### PR DESCRIPTION
**Pull request by marcio-santos-zocdoc**
@Zocdoc/ci-devs 

We need to split the Execute function on two in order to lock only the Database  creation.
Tested locally, Scout tests running green.

### CODE REVIEW

File: model/Command/CreateCommand.cs [hunk 1]
```diff
@@ -5,7 +5,7 @@
 namespace SchemaZen.Library.Command {
     public class CreateCommand : BaseCommand {
 
-        public void Execute(string databaseFilesPath)
+        public void CreateDatabases(string databaseFilesPath)
```
Comment (chris-smith-zocdoc) [inline model/Command/CreateCommand.cs]: Does this create a single database or multiple?

Comment (marcio-santos-zocdoc) [reply model/Command/CreateCommand.cs]: ow Rick, you got me there. It creates a single DB.

Comment (marcio-santos-zocdoc) [reply model/Command/CreateCommand.cs]: Done, Rick.

**Comment by colin-hanson-zocdoc** [comment]
Did you run the schemazen tests locally as well?

**Comment by marcio-santos-zocdoc** [comment]
Yes, @colin-hanson-zocdoc all Schemazen tests working.

File: model/Command/CreateCommand.cs [hunk 2]
```diff
@@ -20,7 +20,19 @@ public void Execute(string databaseFilesPath)
                 throw new InvalidOperationException(msg);
             }
 
-            db.CreateFromDir(Overwrite, databaseFilesPath, Logger.Log);
+            db.CreateDBFromDir(databaseFilesPath, Logger.Log);
+            Logger.Log(TraceLevel.Info, Environment.NewLine + "Database created successfully.");
```
Comment (chris-smith-zocdoc) [inline model/Command/CreateCommand.cs]: This should include the database name

**Comment by chris-smith-zocdoc** [comment]
:shipit:

**Pull request merged**